setup.py

```diff
@@ -43,6 +43,7 @@
     extras_require={
         "optimization": [
             "botorch>=0.10.0",
+            "entmoot>=2.0",
```

Comment:
Let us create an extra `extra` for entmoot, or what do you think? @bertiqwerty? @R-M-Lee 

Perhaps we should do it also for botorch ...

Reply:
Hm after looking again I think another `extra` is probably better, since that's how it's done for `xgb`.